NewBrewer/NewBrewer/Views/SearchBeer/Custom Cell/BeerListTableViewCellModel.swift

```diff
@@ -0,0 +1,46 @@
+//
+//  BeerListTableViewCellModel.swift
+//  NewBrewer
+//
+//  Created by Jake Gloschat on 4/12/23.
+//
+
+import Foundation
+import FirebaseFirestore
+
+protocol BeerListTableViewCellModelDelegate: AnyObject {
+    func favoritesCheckedSuccesfully()
+}
+
+struct BeerListTableViewCellModel {
+    
+    // MARK: - Properties
+    private weak var delegate: BeerListTableViewCellModelDelegate?
+    private var service: FirebaseServicable
+//    var beers: [BeerToSave]
+    let ref = Firestore.firestore()
+    
+    init(delegate: BeerListTableViewCellModelDelegate, firebaseService: FirebaseServicable = FirebaseService()) {
+        self.delegate = delegate
+        self.service = firebaseService
+    }
+    
+//    func checkSavedBeers() {
```

Comment:
remove unnecessary code

Reply:
Again, refactored code to remove logic from the cell to the SearchBeerViewModel so I wasn't doing a call for each cell.  Removed entire TableViewCellModel file.